edx_exams/apps/api/v1/views.py

```diff
@@ -358,28 +358,25 @@ def get_response(cls, exam, user):
         elif exam.due_date is not None and timezone.now() >= exam.due_date:
             data, grant_access, response_status = {}, True, status.HTTP_200_OK
 
-        response = Response(status=response_status,
-                            data=data)
-
         if grant_access:
             log.info("Creating exam access token")
             access_token = sign_token_for(user.lms_user_id, expiration_window, claims)
-            response.set_cookie(
-                settings.ACCESS_TOKEN_COOKIE_NAME, access_token,
-                domain=settings.ACCESS_TOKEN_COOKIE_DOMAIN
-            )
+            data = {ACCESS_TOKEN_COOKIE_NAME: access_token, ACCESS_TOKEN_COOKIE_NAME + "_expiration": expiration_window}
```

Comment:
Added both access token and expiration window to the body of the response. Let me know if there is a better way to do this or if you think the expiration field should be renamed.